test/test_parallel_executor.cpp

```diff
@@ -194,18 +140,16 @@ TEST(ParallelExecutor, ExecuteOnSingleThread) {
 
   std::vector<Tensor> tf_input_tensors{x, y};
   shared_ptr<ngraph::runtime::Executable> ng_exec;
-
+  shared_ptr<PipelinedTensorsStore> pts;
+  std::tuple<int, PipelinedTensorVector, PipelinedTensorVector> io_tensors;
   // Call the Executor to compile the funcion
   bool cache_hit = false;
-  ASSERT_OK(executor.GetNgExecutable(tf_input_tensors, ng_exec, cache_hit));
+  std::string ser_ng_func;
+  ASSERT_OK(executor.GetExecutableFunctionAndTensors(
+      tf_input_tensors, ng_exec, ser_ng_func, pts, cache_hit));
+  io_tensors = pts.get()->get_tensors();
```

Comment:
It seems a few tests have been removed. Have we replaced them? Are they no longer required?

Reply:
There were seperate tests for GetTensorFromPipeline(), which we dont need anymore as this function is merged into GetExecutableFunctionand Tensors() About multi-threading tests, since we are testing multithreading  for data cache, there are suffiecient tests there
We removed locks from executor so not much to testsbBut if anyone has more ideas please let me know